tests/NRedisStack.Tests/Search/SearchTests.cs

```diff
@@ -3159,4 +3159,75 @@ public void TestMultiGeospatial()
         Assert.True(res1.Documents.All(d => d.Id.Equals("doc1")));
     }
 
+
+    public class RawWKTShape : Geo.Shape
```

Comment:
Isn't it more user-friendly to just use something like this, but as an implementation detail for the `Query` method overloads accepting `string` instead of `Shape`?

Something like:
```
public Query AddGeospatial(string property, Functions function, string rawWktShape)
```

Reply:
for sure its more user friendly. i am not strongly against to this kind of overload, just trying to evaluate if we introduce possible future limitations in case we decide to put more responsibility and enrich `Shape` in future. And it doesn't sound so unlikely to me.
Would prefer to stick with the OOP basics.